src/amo/components/StaticPages/About/index.js

```diff
@@ -63,23 +73,21 @@ export class AboutBase extends React.Component {
           <p>{i18n.gettext(`There are plenty of ways to get involved. If you're on the technical side:`)}
             <ul>
               <li>
-                <Link to={'/developers'}>{i18n.gettext('Make your own add-on')}</Link>.&nbsp;
-                {i18n.gettext('We provide free hosting and update services and can help you reach a large audience of users.')}
+                <Link to={'/developers'}>{i18n.gettext('Make your own add-on')}</Link>
+                {i18n.gettext('. We provide free hosting and update services and can help you reach a large audience of users.')}
               </li>
               <li>
-                {i18n.gettext('If you have add-on development experience,')}&nbsp;
-                <a href="https://wiki.mozilla.org/Add-ons/Reviewers/Applying">{i18n.gettext('become an Add-on Reviewer')}</a>!&nbsp;
-                {i18n.gettext('Our reviewers are add-on fans with a technical background who review add-ons for code quality and stability.')}
+                {i18n.gettext('If you have add-on development experience, ')}&nbsp;
```

Comment:
This section should be all one block of `gettext` with the link a substitution via `sprintf` using a start and end placeholder as done elswhere.